docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+# Developer compose file for required services
+# May be used as reference for production use but this file does not contain any best practices
+# This file has no service for the game defined as it will be started from the source code
+
+version: '3'
+services:
+
+  # authentication service
+  auth:
+    image: ghcr.io/meikellp/quantum-core-x/auth
+    restart: unless-stopped
+    ports:
+    - "11002:11002"
+    volumes:
+      - ./Executables/Auth/appsettings.json:/app/appsettings.json:ro
+
+  # migrate service - required once
+  migrate:
+    image: ghcr.io/meikellp/quantum-core-x/migrator
+    restart: no
+    network_mode: host
```

Comment:
Why do we use host network mode here? Would prefer to instead use "db" as a host here in the command.

Reply:
I was unable to do it without the host mode. Feel free to suggest a (working) change here :)